config/schedule.rb

```diff
@@ -3,9 +3,7 @@
 
 # Learn more: http://github.com/javan/whenever
 
-app_config = YAML.load_file(File.join(__dir__, 'application.yml'))
```

Comment:
Whenever is still in use.

I just had a little test and we need to add some loading here or ENV isn't populated.